app/middleware/errors.js

```diff
@@ -12,6 +21,12 @@ module.exports = (app, conf, log) => {
 
   return (err, req, res, next) => {
     errlog.error(err);
+
+    if (err.message.startsWith('state mismatch')) {
```

Comment:
I think this `if` should be inside `handle_state_mismatch`, which should return true if there was a state mismatch that was handled.

Reply:
I'll play with it 👍

Reply:
I played with it a little bit but it made the code a bit more complicated so I'll leave it as it is as discussed offline.